app/views/application_letters/_application_selective.html.erb

```diff
@@ -1,4 +1,4 @@
-<% if can? :edit_applicants, Event and not (application_letter.event.application_status_locked == true) %>
+<% if (can? :edit_applicants, Event) && (application_letter.event.phase == :selection) %>
```

Comment:
I think the selection of participants should only be available in the selection phase (therefore the name) and if the event is in the selection phase the application should not be locked, because otherwise it it not in the selection phase.